kate_hooke/disappearing_ninja/disappearing_ninja.py

```diff
@@ -0,0 +1,24 @@
+from flask import Flask, render_template, request, redirect
+app = Flask(__name__)
+
+@app.route('/')
+def landing_page_home():
+    return render_template('index.html')
+
+@app.route('/ninja')
+def all_ninjas():
+    return render_template('ninja.html')
+
+@app.route('/ninja/blue')
```

Comment:
`/ninja/blue`, `/ninja/red`, and `/ninja/orange` should all be one route with a route parameter for the color: `/ninja/<color>`